service/src/test/java/uk/gov/hmcts/reform/fpl/service/CreateC21OrderServiceTest.java

```diff
@@ -39,12 +40,14 @@
 
     private DateFormatterService dateFormatterService = new DateFormatterService();
     private HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration = new HmctsCourtLookupConfiguration(CONFIG);
+    private DocumentConfiguration documentConfiguration = new DocumentConfiguration("");
```

Comment:
Not sure why this needs to be field if neverused in class (apart from beforeEach)